frontend/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "esModuleInterop": true,
     "outDir": "build",
     "resolveJsonModule": true,
-    "rootDir": "src",
```

Comment:
Details in the commit message for why these settings were changed: https://github.com/ScopeLift/umbra-protocol/pull/304/commits/6610b9a5465cd0729066e4b5b08ebf57f1d55deb